scst-store/deployment-details.hbs.md

```diff
@@ -20,6 +20,40 @@ The installation creates the following in your Kubernetes cluster:
 
 ## <a id='configuration'></a> Deployment configuration
 
+### Supported Network Configurations
+
+VMware recommended connection methods based on Tanzu Application Platform setup:
+
+* Single or multi-cluster with Contour = `Ingress`
+* Single cluster without Contour and with `LoadBalancer` support = `LoadBalancer`
+* Single cluster without Contour and without `LoadBalancer` = Port forwarding
+* Single cluster without Contour, without `LoadBalancer` and user does not have port forwarding access = `NodePort`
+* Multi-cluster without Contour = Not supported
+
+For a production environment, VMware recommends that Supply Chain Security Tools - Store is installed with ingress enabled.
+
+#### <a id='appserv-type'></a>App service type
+
+Supported values include `LoadBalancer`, `ClusterIP`, `NodePort`. The `app_service_type` is set to `LoadBalancer` by default. If your environment does not support `LoadBalancer`, and you want to use `ClusterIP`, configure the `app_service_type` property in your `metadata-store-values.yaml`:
+
+```yaml
+app_service_type: "ClusterIP"
+```
+
+If the `ingress_enabled` property is set to `"true"`, VMware recommends setting the `app_service_type` property to `"ClusterIP"`.
```

Comment:
Do we want to be nice and specifically point out that it's a string and not a boolean value?

Reply:
okay I'll add it to the `Ingress Support` section